.github/actions/with-k-docker/Dockerfile

```diff
@@ -15,9 +16,13 @@ RUN    apt-get -y update    \
          make               \
          python3.10-dev     \
          python3.10-venv    \
-         /llvm-backend.deb  \
-         /kframework.deb    \
-    && apt-get -y clean
+         /kframework.deb
```

Comment:
So IIUC this Docker file is used with two kinds of `kframework.deb`: one that contains the LLVM backend and on that doesn't.